jobs/google_cpi/templates/config/cpi.json.erb

```diff
@@ -22,6 +23,9 @@ params = {
 if_p('google.project') do |project|
   params["cloud"]["properties"]["google"]["project"] = project
 end
+if_p('google.user_agent_prefix') do |user_agent_prefix|
```

Comment:
btw dont need to put into if_p. p('google.user_agent_prefix') is enough since it has a default.